File: src/main/java/com/core/base/corebase/controller/organization/dto/OrganizationRes.java
// Copyright 2025 Kakao Healthcare Corp. All Rights Reserved.

package com.core.base.corebase.controller.organization.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class OrganizationRes {
    private Long id;
    private String name;
    private String logoFileName;
    private String ceo;
    private String telNumber;
    private String address;
    private List<TeamRes> teams;
}

File: src/main/kotlin/com/core/base/corebase/controller/organization/dto/OrganizationRes.kt
package com.core.base.corebase.controller.organization.dto

class OrganizationRes(
    val id: Long,
    val name: String,
    val logoFileName: String?,
    val ceo: String?,
    val telNumber: String?,
    val address: String?,
    val teams: List<TeamRes>?
)